mobile/src/actions/mediaBackup.js

```diff
@@ -33,8 +45,11 @@ export const mediaBackup = (dir) => async (dispatch, getState) => {
     let photos = await getFilteredPhotos()
     const alreadyUploaded = getState().mobile.mediaBackup.uploaded
     const dirID = await getDirID(dir)
+    const totalUpload = photos.length - alreadyUploaded.length
```

Comment:
Precisely it is not the same.

If a user deletes already uploaded photos, they will count in `alreadyUploaded` but not in `photos`, which will distort the calculation.